common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -78,10 +77,21 @@
 
         Sequence.prototype.updatePageTitle = function() {
             // update the page title to include the current section
-            var positionLink = this.link_for(this.position);
+            var positionLink = this.link_for(this.position),
+                currentSectionTitle;
```

Comment:
@ormsbee yup, just declaring -- it's also done this way up here https://github.com/edx/edx-platform/blob/a592699e541857ba4c126a58069a9d0fd3ded89c/common/lib/xmodule/xmodule/js/src/sequence/display.js#L170 in the same file so I figured that style was okay.